libs/shared/data-access/audit-api/src/lib/models/audit-event/audit-event.model.ts

```diff
@@ -97,7 +97,8 @@ export interface EventUpdates {
   oldState: string;
   newState: string;
   label: string;
-  documentId: string | undefined;
+  newDocumentId: string | undefined;
+  oldDocumentId: string | undefined;
```

Comment:
This is the more standard way of doing this in typescript
```suggestion
  newDocumentId?: string;
  oldDocumentId?: string;
```